src/lib/email.js

```diff
@@ -0,0 +1,42 @@
+import nodemailer from 'nodemailer';
+import StudentStore from '../store/student-store';
+
+const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+        user: process.env.GMAIL_USER,
+        pass: process.env.GMAIL_PASSWORD
+    }
+});
+
+export function sendMail(emails) {
+    let bcc = emails.join(',');
+    let subject = 'Tech Support Notification: Overdue Equipment';
+    let body = 'Hello,%0A%0AOur records indicate that you have overdue equipment that was checked out from Tech Support. Please visit us soon to return the equipment.%0A%0AThank you!';
```

Comment:
Agreed, except that this is one mass-email that will be send out, bcc'ing all students with overdue items. To send individual messages would be painful for the person doing it.